UnitTests/Plugins.ResxLocalization.UnitTest/MvvmCross.Plugins.ResxLocalization.UnitTest.csproj

```diff
@@ -19,7 +19,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
@martijn00 is it worth putting a comment next to this include so that it's clear why this include is necessary? In scanning the project files the Service Include elements are one of the few things that just doesn't make sense

Reply:
Not sure. These ones are used in all the unit tests, so i think they are used to run them or something.